LiveStream/Models/LiveStreamEvent.swift

```diff
@@ -0,0 +1,154 @@
+import Argo
+import Curry
+import Runes
+
+public struct LiveStreamEvent: Equatable {
+  public let creator: Creator
+  public let firebase: Firebase
+  public let id: Int
+  public let openTok: OpenTok?
+  public let stream: Stream
+  public let user: User
+
+  public struct Stream {
+    public let backgroundImageUrl: String
+    public let description: String
+    public let hasReplay: Bool
+    public let hlsUrl: String?
+    public let isRtmp: Bool
+    public let isScale: Bool
+    public let liveNow: Bool
+    public let maxOpenTokViewers: Int
+    public let name: String
+    public let projectWebUrl: String
+    public let projectName: String
+    public let replayUrl: String?
+    public let startDate: Date
+    public let webUrl: String
+
+    // Useful for safeguarding against getting a `hasReplay == true` yet the `replayUrl` is `nil`.
+    public var definitelyHasReplay: Bool {
+      return self.hasReplay && self.replayUrl != nil
```

Comment:
🤔 we're not sure! we were seeing some funky data coming back from the live stream api, and so we are just trying to be super explicit.

ostensibly there is a chance that maybe the `replayUrl` exists cause it's basically an archive of the stream, but perhaps the creator didnt want a replay to be available? or there were technical difficulties?

/cc @Jwomers